### PR TITLE
update discussion of how OS is installed

### PR DESCRIPTION
**Pull request by dhellmann**
Signed-off-by: Doug Hellmann <dhellmann@redhat.com>

### CODE REVIEW

File: enhancements/installer/single-node-installation-bootstrap-in-place.md [hunk 1]
```diff
@@ -171,8 +165,27 @@ logic, the modified script will:
...
+host's local storage. It may be necessary to provide other inputs to
+`coreos-install` at the same time (network addresses, etc.), and we
+have not yet completed the investigation and experimentation work
+required to design a supportable API for the complex details that may
+be needed. Therefore, to unblock progress on the bootstrap-in-place
+feature, we will temporarily treat installing the operating system as
+a manual step. Users running the OpenShift installer by hand will need
+to login to the host via `ssh` and run `coreos-install`, passing the
+appropriate arguments. Users relying on integration with Hive and the
+assisted installer for multi-cluster management will not need to
```
Comment (romfreiman) [inline enhancements/installer/single-node-installation-bootstrap-in-place.md:180]: I think we call it central cluster management but not 100% sure

File: enhancements/installer/single-node-installation-bootstrap-in-place.md [hunk 2]
```diff
@@ -469,9 +482,12 @@ In general, we try to use the same stages (alpha, beta, GA), regardless how the
...
 - Available by default
-- Update the installer to replace
-  `OPENSHIFT_INSTALL_EXPERIMENTAL_BOOTSTRAP_IN_PLACE_COREOS_INSTALLER_ARGS`
-  with a field in the `install-config.yaml` schema.
+- Either define a maintainable API for passing arguments to
+  `coreos-install` in the OpenShift installer so that the
+  bootstrapping services can install the operating system on the host,
+  or definitively decide that installing the operating system should
+  always be a manual step when the user is running the OpenShift
+  installer outside of our multi-cluster management integration.
```
Comment (romfreiman) [inline enhancements/installer/single-node-installation-bootstrap-in-place.md:490]: central..

**Comment by romfreiman** [comment]
Looks good!

File: enhancements/installer/single-node-installation-bootstrap-in-place.md (hunk 1 above)
Comment (dhellmann) [reply enhancements/installer/single-node-installation-bootstrap-in-place.md:180]: OK, I thought the term we've been using was "multi-cluster" but I'm happy to change it.

**Pull request merged**